assets/images/icon-close.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <!-- Generator: Sketch 53.2 (72643) - https://sketchapp.com -->
+  <title>Icon/32/Close   </title>
+  <desc>Created with Sketch.</desc>
+  <g id="Icon/32/Close---" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+    <polygon id="Path" fill="#000000" points="29 5.73378076 26.2662192 3 16 13.256525 5.73378076 3 3 5.73378076 13.256525 16 3 26.2662192 5.73378076 29 16 18.743475 26.2662192 29 29 26.2662192 18.743475 16"></polygon>
+  </g>
+</svg>
```

Comment:
Any particular reason for using our own image instead of the one provided by `@asc-assets`?

Reply:
I didn't know @asc-assets existed. And the amsterdam-styled-component site does not function properly anymore, so it is quite difficult to figure out the options of each component.